src/lib/components/column-profile/ColumnProfileDetails.svelte

```diff
@@ -16,11 +19,23 @@
   export let type;
   export let summary;
   export let totalRows;
+  export let name;
+  export let entityId;
   export let containerWidth: number;
 
   export let indentLevel = 1;
 
   export let active = false;
+
+  // Make sure priority is updated in case the profile is already opened
+  afterUpdate(async () => {
+    if (active) {
+      dataModelerService.dispatch("updateColumnPriority", [entityId, name]);
+    }
+  });
+  // $: if (active) {
+  //   dataModelerService.dispatch("updateColumnPriority", [entityId, name]);
+  // }
```

Comment:
@AdityaHegde @hamilton Is there a better way to achieve the same with minimal re-renders? Using `$: if(active)` does not execute if the profile is already active

Reply:
We should call this where active is actually changed.

Reply:
agreed, would be better to have this call not as a reactive statement but where there is a click callback

Reply:
The issue with having it in the callback is that when a profile is already opened and the user changes the query, the `active` callback is not executed thus no action is dispatched to prioritize that column